docs/topics/faq.md

```diff
@@ -96,7 +96,7 @@ When targeting native, Kotlin will produce platform-specific code (via LLVM).
 
 Kotlin lets you choose the version of JVM for execution. By default, the Kotlin/JVM compiler produces Java 6 compatible bytecode.
 If you want to make use of optimizations available in newer versions of Java, you can explicitly specify the target Java
-version from 8 to 15. Note that in this case the resulting bytecode might not run on lower versions. 
+version from 8 to 16. Note that in this case the resulting bytecode might not run on lower versions. 
```

Comment:
```suggestion
version from 9 to 16. Note that in this case the resulting bytecode might not run on lower versions. 
```
(since 8 is the default... But maybe it's worth rephrasing the whole sentence, I'm not sure.)